src/eradiate/cli.py

```diff
@@ -292,5 +305,204 @@ def info(data_stores, list_registry):
                 console.print(textwrap.indent(reprs[repr_key], " " * 4))
 
 
+@main.group()
+def srf():
```

Comment:
Please move to a dedicated module (see updated CLI code on `main`, which you'll have to rebase on).